api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -66,6 +69,9 @@
 	private UserService userService;
 
 	private MessageSourceService messages;
+	
+	@Autowired
+	private UserDAO dao;
 
```

Comment:
Where is the test that ensures that if the key is expired, then you get back nothing